src/main/java/parser/Parser.java

```diff
@@ -37,125 +36,24 @@ public Command getUserCommand(String userInput) throws TantouException {
         }
 
         return switch (inputList[COMMAND_INDEX]) {
-            case BYE_COMMAND -> new ByeCommand();
-            case CATALOG_COMMAND ->
-                    processCatalogCommand(trimmedUserInput);
-            case VIEW_COMMAND ->
-                    processViewCommand(trimmedUserInput);
-            case SALES_COMMAND ->
-                    processAddSalesCommand(trimmedUserInput);
-            default ->
-                    throw new TantouException("Invalid command provided!");
+        case BYE_COMMAND -> new ByeCommand();
+        case CATALOG_COMMAND ->
+                processCatalogCommand(trimmedUserInput);
+        case VIEW_COMMAND ->
+                processViewCommand(trimmedUserInput);
+        case SALES_COMMAND ->
+                processAddSalesCommand(trimmedUserInput);
+        default ->
+                throw new TantouException("Invalid command provided!");
         };
```

Comment:
I'm not sure I like how this is formatted. Perhaps store the result of the `switch` block first before returning it. This should help with potential formatting issues.